### ISSUE TITLE
Physics-guided neural networks for feedforward control: From consistent identification to feedforward controller design. (arXiv:2204.00431v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#17501]
Physics-guided neural networks for feedforward control: From consistent identification to feedforward controller design. (arXiv:2204.00431v1 [eess.SY])<br>
https://ift.tt/vwcJGqt<br>
<p>Model-based feedforward control improves tracking performance of motion systems, provided that the model describing the inverse dynamics is of sufficient accuracy. Model sets, such as neural networks (NNs) and physics-guided neural networks (PGNNs) are typically used as flexible parametrizations that enable accurate identification of the inverse system dynamics. Currently, these (PG)NNs are used to identify the inverse dynamics directly. However, direct identification of the inverse dynamics is sensitive to noise that is present in the training data, and thereby results in biased parameter estimates which limit the achievable tracking performance. In order to push performance further, it is therefore crucial to account for noise when performing the identification. To address this problem, this paper proposes the use of a forward system identification using (PG)NNs from noisy data. Afterwards, two methods are proposed for inverting PGNNs to design a feedforward controller for high-precision motion control. The developed methodology is validated on a real-life industrial linear motor, where it showed significant improvements in tracking performance with respect to the direct inverse identification.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
